packages/host/tests/integration/components/computed-test.gts

```diff
@@ -587,5 +587,108 @@ module('Integration | computeds', function (hooks) {
       .containsText('Scarsdale');
   });
 
-  skip('can render a computed linksTo relationship');
+  test('can render a computed linksTo relationship', async function (assert) {
+    let { field, contains, linksTo, Card, Component } = cardApi;
+    let { default: StringCard } = string;
+    class Pet extends Card {
+      @field name = contains(StringCard);
+      static embedded = class Embedded extends Component<typeof this> {
+        <template>
+          <span data-test='name'><@fields.name /></span>
+        </template>
+      };
+    }
+    class Person extends Card {
+      @field firstName = contains(StringCard);
+      @field bestFriend = linksTo(Pet);
+      static embedded = class Embedded extends Component<typeof this> {
+        <template>
+          <span data-test='firstName'><@fields.firstName /></span>
+          <span data-test='bestFriend'><@fields.bestFriend /></span>
+        </template>
+      };
+    }
+    class Post extends Card {
+      @field title = contains(StringCard);
+      @field author = contains(Person);
+      @field friend = linksTo(Pet, {
+        computeVia: function (this: Post) {
+          return this.author.bestFriend;
```

Comment:
if it possible that we can return the card ID in these computeds instead of deserialized objects?